### DISCUSSION TITLE
Kudos on Keyboard shortcuts for mark mode!

### DISCUSSION THREAD

**Discussion by jsteinbergTCALLC** [microsoft/terminal#16727] [General]
# Description of the new feature/enhancement

I just want to let your team know that I am loving the keyboard shortcut for mark mode in terminal. I know that this is not a recent addition - but I was an early adopter of Window Terminal, and abandoned it immediately as it did not support keyboard mark/copy/paste - lack of keyboard support for this just killed it for me.. I just recently checked back in, and found that this is now available, and it is great! 

# Proposed technical implementation details (optional)

Give yourselves a pat on the back.

**Comment by DHowett** [comment]
Aw, thanks so much! It means a lot to the team to get nice feedback like this. 😄
